tests/e2e/support.py

```diff
@@ -65,19 +65,30 @@ def create_namespace(self):
         return RuntimeError(e)
 
 
-def create_new_resource_flavor(self):
-    self.resource_flavor = f"test-resource-flavor-{random_choice()}"
-    create_resource_flavor(self, self.resource_flavor)
+def create_new_resource_flavor(self, num_flavors=2):
```

Comment:
@Ygnas Wondering does multiple resource flavours , cluster queues and local queues creation is required for all the tests or this scenario should be covered particularly for heterogeneous test, because this change is impacting for all the sdk tests of kueue setup

Reply:
You are correct these would not be required for the rest of the tests, will update